portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```diff
@@ -84,10 +89,20 @@ private String convertToJsonUsingGson(List<String> comments) {
 
     private final String comment;
```

Comment:
This field has the same name with the class, which can be a bit confusing at some appearances. Can we rename the field to something else to resolve the conflict?

Reply:
I have renamed comment to message.